src/Mutant/MutantExecutionResultFactory.php

```diff
@@ -86,7 +85,7 @@ private function retrieveProcessOutput(Process $process): string
 
     private function retrieveDetectionStatus(MutantProcess $mutantProcess): string
     {
-        if (!$mutantProcess->getMutant()->isCoveredByTest()) {
+        if (!$mutantProcess->getMutation()->hasTests()) {
```

Comment:
I still not convinced by this change. It is confusing IMO (`isCoveredByTest()` -> `hasTests()`).

Reply:
Hm thought I reverted it